npc/re/merchants/shopping_boards.txt

```diff
@@ -0,0 +1,29 @@
+//===== rAthena Script =======================================
+//= Shopping Boards
+//===== Description: =========================================
+//= Shopping Boards work like Bargain Hunter's Catalogues.
+//= They are found in towns and search exclusively in their respective maps.
```

Comment:
If it is only on the same map, then we need to adjust searchstore command?

Reply:
You are right!
I opened a store in geffen to test, and I used the Board in izlude, and the geffen store appeared.

Reply:
Actually searchstore should only work for same map, so our implementation of the command may be wrong XD

Reply:
Oh I was under the assumption as well that the command was already map bound!

Reply:
Same here, agreed that the command must be updated or the script must be changed...

Reply:
I've fixed it on #8456